src/Stratis.Bitcoin/FullNode.cs

```diff
@@ -247,6 +247,20 @@ private void StartPeriodicLog()
             startAfter: TimeSpans.FiveSeconds);
 
             this.Resources.Add(periodicLogLoop);
+
+            IAsyncLoop periodicBenchmarkLoop = this.AsyncLoopFactory.Run("PeriodicBenchmarkLog", (cancellation) =>
+            {
```

Comment:
Do we not want to restrict bench perf to IBD?

Reply:
Yeah, good idea.
fixed